web/modules/custom/usagov_ssg_postprocessing/src/EventSubscriber/PublishedPagesSubscriber.php

```diff
@@ -118,7 +125,7 @@ public function modifyHtml(ModifyHtmlEvent $event) {
         $nodeIDElement = array_search("Page ID", $csv[0]);
         $languageElement = array_search("Taxonomy Level 1", $csv[0]);
         foreach ($csv as $key => $line) {
-          if ($line[$nodeIDElement] == $decoded["nodeID"]) {
+          if (!empty($line[$nodeIDElement]) && ($line[$nodeIDElement] == $decoded["nodeID"] || $line[$nodeIDElement] == 't_' . $decoded["taxonomyID"])) {
```

Comment:
I am getting a whole lot of warnings about this line. Big batches of:  
```
[warning] Undefined array key "taxonomyID" PublishedPagesSubscriber.php:128
```
... and 
```
[warning] Undefined array key "nodeID" PublishedPagesSubscriber.php:128
```

Reply:
Ah I was able to reproduce and re-run. I believe I have fixed the issue, though, I am seeing one and only one potential "duplicate" in the report for the home-page - one home page in English and one in Spanish. Can you refresh my memory if this is expected?
I have updated this branch.